### PR TITLE
ALBS-8 add some basic tests for decode_JWT_token and BuildDoneArtifact.is_debuginfo

### PR DESCRIPTION
**Pull request by monkrobot**

### CODE REVIEW

**Review by anfimovdm — APPROVED** [review]

**Review by JohnKepplers — APPROVED** [review]

File: tests/test_schemas/build_node_schema_test.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+class TestBuildNodeSchema(unittest.TestCase):
+    
+    def test_is_debuginfo(self):
+        test_BuildDoneArtifact = build_node_schema.BuildDoneArtifact(
+            name='test-debuginfo-1',
+            type='build_log',
+            href='http://127.0.0.1:8080/test_example.com'
+        )
+        message = "is_debuginfo don't work with correct data"
+        self.assertEqual(True, test_BuildDoneArtifact.is_debuginfo, message)
```
Comment (soksanichenko) [inline tests/test_schemas/build_node_schema_test.py:15]: self.assertTrue

File: tests/test_schemas/build_node_schema_test.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+    def test_false_is_debuginfo(self):
+        test_BuildDoneArtifact = build_node_schema.BuildDoneArtifact(
+            name='test-debuginfo',
+            type='build_log',
+            href='http://127.0.0.1:8080/test_example.com'
+        )
+        message = "is_debuginfo don't work with incorrect data"
+        self.assertEqual(False, test_BuildDoneArtifact.is_debuginfo, message)
```
Comment (soksanichenko) [inline tests/test_schemas/build_node_schema_test.py:24]: The same as above

**Review by soksanichenko — CHANGES_REQUESTED** [review]

File: requirements.txt [hunk 3]
```diff
@@ -8,4 +8,4 @@ aiohttp==3.7.4.post0
 PyJWT==2.1.0
 alembic==1.7.3
 aioredis==2.0.0
-psycopg2-binary==2.9.1
```
[the remark this answers is not in the document]
Comment (Korulag) [reply requirements.txt:11]: No it's not, we need it for test scheduler

File: tests/test_crud/crud_test.py [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+        message = "Tasks aren't pinged"
+        self.assertEqual(response.status_code, 200, message)
+
+
+class TestCreatePlatform(unittest.TestCase):
+
+    client = TestClient(app)
+    token = jwt_utils.generate_JWT_token({'user_id': 1}, settings.jwt_secret, 'HS256')
+
+    async def custom_db(self):
```
Comment (Korulag) [inline tests/test_crud/crud_test.py:60]: Create a base class with this method derived from TestCase, then inherit it for other tests that need database.

**Review by Korulag — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 3 above)
[the remark this answers is not in the document]
Comment (AvoidMe) [reply requirements.txt:11]: It's duplicate, we have psycopg2-binary two times in out requirements.txt:
```
fastapi[all]==0.68.1
uvicorn==0.13.4
asyncpg==0.24.0
psycopg2-binary==2.9.1        # first one
pydantic==1.8.2
SQLAlchemy==1.4.25
aiohttp==3.7.4.post0
PyJWT==2.1.0
alembic==1.7.3
aioredis==2.0.0
psycopg2-binary==2.9.1        # second
jmespath==0.10.0
```

**Review by rpisarev — COMMENTED** [review]
I guess you should resolve conflicts as well

**Review by rpisarev — APPROVED** [review]

**Review by rpisarev — APPROVED** [review]

**Review by soksanichenko — APPROVED** [review]

**Pull request merged**